tests/HUBViewControllerTests.m

```diff
@@ -1470,6 +1472,27 @@ - (void)testSavingAndRestoringBodyComponentUIState
     XCTAssertEqual(self.component.numberOfReuses, (NSUInteger)2);
 }
 
+- (void)testViewControllerDelegateIsNotifiedWhenComponentIsReused
+{
+    self.contentOperation.contentLoadingBlock = ^(id<HUBViewModelBuilder> viewModelBuilder) {
+        [viewModelBuilder builderForBodyComponentModelWithIdentifier:@"one"].title = @"One";
+        return YES;
+    };
+
+    [self simulateViewControllerLayoutCycle];
+
+    id<UICollectionViewDataSource> const collectionViewDataSource = self.collectionView.dataSource;
```

Comment:
You don't need this as you're only accessing the data source once. Can just inline `self.collectionView.dataSource` on row `1487`.